src/shaders/pbr.vert

```diff
@@ -38,7 +38,7 @@ uniform highp mat3 TextureMatrix
 #endif
 
 void main() {
-  vertexPositionInCamera = ModelViewMatrix * vertexPosition;
+  vec4 vertexPositionInCamera = ModelViewMatrix * vertexPosition;
```

Comment:
Hah. That reminds me -- you know what this PR (and #840) is absolutely missing? *Tests.*

I don't think you need to go extra fancy compared to how massively I'm testing shaders in Magnum, but at least:

1. have a test that tries to compile the shader with all legal flag combinations and verifies all those work without any compilation error (see e.g. [PhongGLTest](https://github.com/mosra/magnum/blob/c226bc2d630f886969509e5a7b13d0a8513c6c70/src/Magnum/Shaders/Test/PhongGLTest.cpp#L596-L617) in Magnum for a quick nonverbose way how to do that)
2. comparing a render of some PBR model to *at least one* ground truth image. This could use Magnum's [CompareImage](https://doc.magnum.graphics/magnum/classMagnum_1_1DebugTools_1_1CompareImage.html) functionality, which is already used in e.g. SimTest I think -- look there for how it's used. When you write the test, run it with `--save-diagnostics some/path` to get or update the ground truth images.

Reply:
If it is not in this PR, it will happen in the following PRs.